Source/UI/NewUI/BillingForm/Style/Default Implmentation/BillingForm/Fields/AddressLine1/DefaultFormStyleAddressLine1Cell.swift

```diff
@@ -0,0 +1,23 @@
+import UIKit
+
+//Address line 1
+struct DefaultFormStyleAddressLine1Cell : BillingFormTextFieldCellStyle {
```

Comment:
Do you have some custom formatting applied locally? I am familiar with a space after comment tags (`// comment`), as well as no space after object declaration when inheriting or conforming (`struct MyCell: BillingFormTextFieldCellStyle`), but your code is consistently the other way around 😄.

Reply:
This is not technically a cell, its a configuration object that will be applied to a cell. As such, the naming is getting very long very quickly, is the enum option I proposed separately an option?

Reply:
I think, no need for cell dequeue for this case. let's discuss more after vacation 
https://www.codementor.io/@warrenburton/do-we-need-to-dequeue-tableview-cells-w8szp55uq

Reply:
Sounds great, thank you for taking the time while you're away @ehab-al-cko . 

Not actionable now, but want to make sure I don't forget, that article is perfectly valid and I know I've asked myself many times that question, so getting someone to build a project and compare it really is useful. However, specifically for our use case, I would still be inclined to use the better performance method as we have no control over what that application does. For all intents and purposes, imagine app using our SDK is doing crypto mining... Any reasonable easy to gain performance improvement on our side will allow them to keep more resources available. Not saying we should over optimise early (I am quite against that as well), but if the standard practice carries an optimisation benefit (however big/small), then that standard practice should be the choice for an SDK).